static/js/github-success.js

```diff
@@ -0,0 +1 @@
+window.close();
```

Comment:
This is because we have an awkward way of getting the token.
Since the apps hosted inside a Iframe on Jira we can't Oauth directly (Oauth/Github does not approve doing things in iframes) so we open a new tab, instead of redirecting to the create-branch(which is an option) we close the tab after succesful token generation then the original page redirects. this stops having 5000000 new tabs and keeps us inside the Jira iframe